src/witan/workspace/components/peer.clj

```diff
@@ -0,0 +1,31 @@
+(ns witan.workspace.components.peer
+  (:gen-class)
+  (:require [org.httpkit.client           :as http]
+            [com.stuartsierra.component   :as component]
+            [taoensso.timbre              :as log]))
```

Comment:
if you are going to line up the :as then you need to fix it each time (I think these are a bit too far over)